src/bmsg.c

```diff
@@ -585,6 +601,73 @@ int main (int argc, char *argv [])
             die ("unknown type %s", argv[argn]);
         }
     }
+    else
+    if (streq (bmsg_command, "request")) {
+
+        char *agent_name = argv[argn];
+        if (!agent_name)
+            die ("missing agent name", NULL);
+
+        argn ++;
+        char *subject = argv[argn];
+        if (!subject)
+            die ("missing subject", NULL);
+
+        mlm_client_set_producer (client, subject);
+
+        zmsg_t *msg = zmsg_new ();
+        zmsg_addstr (msg, "GET");
```

Comment:
Adding "GET" in the payload is a mistake, because it is dependant to the protocol itself